src/applications/resources-and-support/components/ResourcesAndSupportSearchApp.jsx

```diff
@@ -98,11 +97,10 @@ const ResourcesAndSupportSearchApp = () => {
     <div className="usa-grid usa-grid-full">
       <div className="usa-content vads-u-margin-bottom--0 medium-screen:vads-u-margin-bottom--3">
         {errorMessage && (
-          <AlertBox
-            headline="Something went wrong"
-            status="error"
-            content={errorMessage}
-          />
+          <va-alert status="error">
+            <h3 slot="headline">"Something went wrong"</h3>
```

Comment:
`Something went wrong` probably should not have quotes.

Reply:
good catch, ty